<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>455</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="login">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(30, 30, 30);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QFrame" name="login_2_Frame">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>300</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
	background-color: rgb(40, 40, 40);
	border-radius: 5px;
}
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLineEdit" name="emailEdit">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>190</y>
            <width>170</width>
            <height>30</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	border-radius: 3px;
	background-color: rgb(50, 50, 50);
	color: rgb(126, 126, 126)
}
QLineEdit:hover{
	border: 2px solid rgb(20, 20, 20);
}
QLineEdit:focus{
	border: 1px solid rgb(255, 245, 140);
	color: rgb(244, 244, 244);
}</string>
          </property>
          <property name="placeholderText">
           <string>  Email</string>
          </property>
         </widget>
         <widget class="QPushButton" name="okButton">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>240</y>
            <width>120</width>
            <height>20</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border-radius: 3px;
	background-color: rgb(57, 57, 57);
	color: rgb(157, 157, 157);
}

QPushButton:hover{
	border: 2px solid rgb(90, 90, 90);
	background-color: rgb(80, 80, 80);
	color: rgb(33, 33, 33);
}

QPushButton:focus{
	background-color: rgb(255, 245, 140);
	color: rgb(33, 33, 33);
}
</string>
          </property>
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
         <widget class="QLabel" name="loginLabel">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>120</y>
            <width>131</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Arial Rounded MT Bold</family>
            <pointsize>28</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(254, 165, 223);</string>
          </property>
          <property name="text">
           <string>Login</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="loginLabel_2">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>211</width>
            <height>91</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
